SummerProject/config/urls.py

```diff
@@ -19,6 +19,7 @@
 
 urlpatterns = [
     path('admin/', admin.site.urls),
+    path('api/v1/category/', include('categories.urls')),
```

Comment:
Let's rename `category` to `categories`. See https://restfulapi.net/resource-naming/ for explanation why we should rename it.

Reply:
done